src/main/java/net/mcreator/ui/blockly/BlocklyEditorType.java

```diff
@@ -19,30 +19,13 @@
 
 package net.mcreator.ui.blockly;
 
-public enum BlocklyEditorType {
+public record BlocklyEditorType(String folder, String translationKey, String extension, String startBlockName) {

```

Comment:
I suggest merging renaming `folder` to `registryName` and using same name for `translationKey` too, so removing `translationKey` argument here

Reply:
I thought to that, but I thought it was better to keep as many translations as possible.